mods/parsoid.js

```diff
@@ -784,7 +784,8 @@ PSP.makeTransform = function(from, to) {
 
     return function(restbase, req) {
         var rp = req.params;
-        if (!req.body || !req.body[from]) {
+        if ((!req.body && req.body !== '')
```

Comment:
I'm not sure I'm right, but with spec-based request validator in place we might me able to delete this check completely.